src/test/java/uk/gov/ch/developer/docs/ApplicationVariables.java

```diff
@@ -3,8 +3,8 @@
 public interface ApplicationVariables {
 
     String HOME_PATH = "/dev-hub";
-    String HOME_VIEW = "dev-hub/home";
-    String GETTINGSTARTED_PATH = "/dev-hub/getting-started";
-    String GETTINGSTARTED_VIEW = "dev-hub/gettingStarted";
-    String BADREQUEST_PATH = "bad-path";
+    String HOME_VIEW = "dev-hub/overview";
+    String GETSTARTED_PATH = "/dev-hub/getstarted";
+    String GETSTARTED_VIEW = "dev-hub/getStarted";
```

Comment:
Please refactor to include underscores between each variable name for consistency.

Why is this an interface and not a final class?

Reply:
Because we implemented it that way in the first place. Do not remember.